xml/src/test/java/com/baeldung/xml/XMLDocumentWriterUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.xml;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+public class XMLDocumentWriterUnitTest {
+
+    @Test
+    public void givenXMLDocumentWhenWriteIsCalledThenXMLIsWrittenToFile() throws Exception {
+        Document document = createSampleDocument();
+        new XMLDocumentWriter().write(document, "company_simple.xml", false, false);
```

Comment:
My only concern here is that these tests will leave artifacts on the filesystem, right? Can we clean them up in an `@After` or similar?

Reply:
Added cleanUp method